taxi/views.py

```diff
@@ -87,3 +87,33 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+@login_required
+def update_drivers(request, pk: int):
+    car = Car.objects.get(id=pk)
+    if request.user not in car.drivers.all():
+        car.drivers.add(request.user)
+        car.save()
+    else:
+        car.drivers.remove(request.user)
+        car.save()
+
+    context = {"car": car}
+
+    return render(request, "taxi/car_detail.html", context=context)
```

Comment:
It's better to redirect user to car detail view in this case instead of rendering the template again.